.eslintrc.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
 		"Battle": true, "Pokemon": true, "BattleSound": true, "BattleTooltips": true,
 		"BattleAbilities": false, "BattleAliases": false, "BattleBackdrops": false, "BattleBackdropsFour": false, "BattleBackdropsThree": false, "BattleEffects": false,
 		"BattleFormats": false, "BattleFormatsData": false, "BattleLearnsets": false, "BattleItems": false, "BattleMoveAnims": false, "BattleMovedex": false, "BattleNatures": false, 
-		"BattleOtherAnims": false,  "BattlePokedex": false,"BattlePokemonSprites": false, "BattlePokemonSpritesBW": false, "BattleSearchCountIndex": false, "BattleSearchIndex": false,
+		"BattleOtherAnims": false,  "BattlePokedex": false,"BattlePokemonSprites": true, "BattlePokemonSpritesBW": true, "BattleSearchCountIndex": false, "BattleSearchIndex": false,
```

Comment:
That the variable is modified in one of the linted files.
Really, it means that the linter is able to figure out that it's a global assign for only these cases, as the rest of them are via the ``exports`` alias for ``window``.
We might as well use ``exports`` for ``BattlePokemonSprites*``...